server/src/util.js

```diff
@@ -0,0 +1,65 @@
+const path = require('path');
+const YAML = require('yamljs');
+
+/**
+ * Fetches the input file extension
+ * @param {object} file - The input file received as form-data
+ * @returns {string} The extension of the input file
+ */
+function getFileExtension(file) {
+  if(!file.originalname) return null;
+  return path.parse(file.originalname).ext;
+}
+
+/**
+ * Parses the input specification
+ * @param {object} file - The input file received as form-data
+ * @returns {object|string} The JSON for the input specification
+ */
+function parseInputFile(file) {
+  const fileExt = getFileExtension(file);
+  let origFile = Buffer.from(file.buffer).toString();
+  if(fileExt === '.json') {
+    origFile = JSON.parse(origFile);
+    return origFile;
+  }
+  else if(fileExt === '.yaml') {
+    origFile = YAML.parse(origFile);
+    return origFile;
+  }
+  else {
+    return fileExt ? 'invalid' : null;
+  }
+}
+
+/**
+ * Sends response based on the requirement
+ * @param {object} result - The output file received from lucy's converter
+ * @param {object} file - The input file received as form-data
+ * @param {string} syntax - The expected syntax of the target specification
+ * @param {boolean} toFile - Whether the response should have the specification file as an attachment
+ * @param {object} response - The response parameter received by the callback of the express endpoint
+ * @returns {object} - The response object that is sent to the client
+*/
+function sendResponse(result, file, targetSyntax, toFile, response) {
```

Comment:
this is why unnecessary comments are more problem - here in the comment you say param as syntax and in the params you're calling it targetSyntax. Clean up these comments and name your variables better instead. @pcgamer1